test-data/unit/check-flags.test

```diff
@@ -194,7 +194,7 @@ x = 0  # type: NoReturn  # E: Incompatible types in assignment (expression has t
 [builtins fixtures/dict.pyi]
 
 [case testNoReturnImportFromTyping]
-from typing import NoReturn
```

Comment:
BTW, you mean `typing.NoReturn` is in cpython master right? It's not yet in 3.6.1. I think it's fine for mypy to allow things that are valid on cpython master.